### DISCUSSION TITLE
Dotted border arround the writing line in input

### DISCUSSION THREAD

**Discussion by IttzyTT** [JedWatson/react-select#4929] [Q&A]
I get this strange border around the text i write, how do i remove it?

![Skärmavbild 2021-11-24 kl  15 35 48](https://user-images.githubusercontent.com/70379450/143258048-fc3ada63-1deb-477d-860e-e24e65eb08ef.png)

**Comment by ebonow** [comment]
There is likely a style in your implementation applying an outline on the focused input.

**Reply by IttzyTT** [reply]
ints not on the input its the border around the text "why this border", inside of the input

**Reply by ebonow** [reply]
You also titled this **Dotted border arround the writing line in input** but it's obviously a **solid line** style and if you were confused on dotted vs solid, then its also likely you'd be confused by a border vs outline. 

Don't believe me? Tell me which one has the border and which has the outline.
![Screen Shot 2021-11-24 at 7 26 56 AM](https://user-images.githubusercontent.com/1007579/143266831-64828567-a923-4495-bbbc-ac795aeed328.png)

If you want to put your snark aside instead of trying to contradict my speculation on a screenshot of a broken implementation in which no code was provided, you could simply inspect element and find out where the styling is coming from and answer your own question.

From my experience though, the input (aka: the writing line) can sometimes have a blue focus ring applied to it when the input is focused (for accessibility reasons) from having other styling libraries applied to it (ie: tailwind). It is also the default behavior of many browsers in which react-select overrides to give the app more cohesion, so it makes sense to believe something is overriding the styling provided by react-select.

Here is the codesandbox if you want to have some indication on how to apply styling to the input component.
https://codesandbox.io/s/aged-water-nw8nl?file=/src/App.js
